cosmwasm/contracts/pyth/src/msg.rs

```diff
@@ -43,7 +48,17 @@ pub enum ExecuteMsg {
 #[serde(rename_all = "snake_case")]
 pub struct MigrateMsg {}
 
-pub use pyth_sdk_cw::{
-    PriceFeedResponse,
-    QueryMsg,
-};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
```

Comment:
I've copied all the messages from the sdk into the code here so I can iterate quickly. I'll go back and update the SDK once the contract is done.

Reply:
Or remove the sdk entirely :D